ENH: callout section for comments on static blog (#194)
carpentries-incubator/jekyll-pages-novice: _episodes/arrays.md

> >
> {: .solution }
{: .challenge }

> ## Comments on a static blog
>

> > Comments section can be included in a static blog using one of the below approaches.

Reviewer: ```suggestion
> When setting up a blog on a website, it is common to want to include a comment feed 
> as a way for readers to respond to the content of a post, direct questions to the author, etc.
> A comments section can be included in a static blog using one of the below approaches.
```

A little more context to introduce this callout would be good.

> > 1. Disqus is the first approach. Disqus website offers [instructions](https://disqus.com/admin/install/platforms/jekyll/) on how to enable comments for Jekyll platform (used for GitHub pages). [Demo](https://erresen.github.io/jekyll/disqus/2016/08/26/getting-comments-working-jekyll-disqus.html). Although if you are privacy inclined, the likes of Privacy Badger blocks Disqus widget and might look for alternatives.

Reviewer: ```suggestion
> 1. Disqus is the first approach. The Disqus website offers 
>    [instructions on how to enable comments for Jekyll platform (used for GitHub pages)](https://disqus.com/admin/install/platforms/jekyll/). 
>    [Here is a demo of Disqus with Jekyll](https://erresen.github.io/jekyll/disqus/2016/08/26/getting-comments-working-jekyll-disqus.html). 
>    Although if you are privacy inclined, browser extensions such as Privacy Badger block the Disqus widget so you might look for alternatives.
```

Split source text across multiple lines, and expand the description a little bit e.g. many readers may not know what Privacy Badger is.

Reviewer: I have expanded the link text in each of these points, [to make the links more accessible e.g. for users visiting the site with a screen reader](https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/#toc_Write-helpful-link-text).

Author: agreed Toby, great suggestion!

> > 2. [Staticman](https://staticman.net/) is the second approach. This setup is tedious and it is easy to get the configuration wrong. [Demo](https://travisdowns.github.io/blog/2020/02/05/now-with-comments.html).

Reviewer: ```suggestion
> 2. [Staticman](https://staticman.net/) is the second approach. 
>    This setup is tedious and it is easy to get the configuration wrong. 
>    [Here is a demo of comments with Staticman](https://travisdowns.github.io/blog/2020/02/05/now-with-comments.html).
```

Split source text across multiple lines.

> > 2. [Staticman](https://staticman.net/) is the second approach. This setup is tedious and it is easy to get the configuration wrong. [Demo](https://travisdowns.github.io/blog/2020/02/05/now-with-comments.html).

Reviewer: I would prefer to use more passive language than "tedious" here. Perhaps we could go with something like below?

> The setup is more complex than some of the other options and, in our experience, it is easy to get the configuration wrong.

Author: agreed, fixed with 87c58ae

> > 3. [Just Comments](https://just-comments.com/) is the third approach. [Demo](https://60devs.com/adding-comments-to-your-jekyll-blog.html).

Reviewer: ```suggestion
> 3. [Just Comments](https://just-comments.com/) is the third approach. 
>    [Here is a demo of comments with Just Comments](https://60devs.com/adding-comments-to-your-jekyll-blog.html).
```
Split source text across multiple lines.

> > 4. [fastpages](https://github.com/fastai/fastpages) is the most recent addition to enable comments via GitHub comments (uses [Utterances](https://utteranc.es/)). fastpages comes with lots of options and is used by several researchers. [Demo](https://fastpages.fast.ai/fastpages/jupyter/2020/02/21/introducing-fastpages.html).

Reviewer: ```suggestion
> 4. [fastpages](https://github.com/fastai/fastpages) is the most recent addition to 
>    enable comments via GitHub comments (uses [Utterances](https://utteranc.es/)). 
>    fastpages comes with lots of options and is used by several researchers. 
>    [Here is a demo of comments with fastpages](https://fastpages.fast.ai/fastpages/jupyter/2020/02/21/introducing-fastpages.html).
```

Split source text across multiple lines.

> From the comments section implementation standpoint, Disqus looks most simple followed by Just Comments and Staticman.

> > Fastpages is much more than enabling a comments section. It's a blog template that uses Jekyll as a base, improves on Jekyll experience and offers additional features. For researchers this may be the option to go.

Reviewer: ```suggestion
> Fastpages is much more than enabling a comments section. 
> It is a blog template that uses Jekyll as a base, improves on Jekyll experience and offers additional features. 
> For researchers this may be the option to go.
```

Split source text across multiple lines.
